### PR TITLE
[chore] eslint-tailwind

### PR DESCRIPTION
**Pull request by zakrzaq**
eslint-tailwind
custom class names enabled
lint & fix

### CODE REVIEW

File: components/Product/Tile.vue [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <template>
   <nuxt-link
-    class="group block hover:shadow-2 hover:bg-white transition-all duration-200"
+    class="hover:shadow-2 group block transition-all duration-200 hover:bg-white"
```
Comment (WawrzenskiPatryk) [inline components/Product/Tile.vue:3]: It's interesting that it didn't group up `hover:`s together 🤔

**Review by WawrzenskiPatryk — COMMENTED** [review]
For some reason this really seems more readable this way, nice 🚀

**Review by elmir-ismayilzada — APPROVED** [review]
Looks really nice!

**Review by WawrzenskiPatryk — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "vue-tsc": "1.0.24"
   },
   "dependencies": {
+    "eslint-plugin-tailwindcss": "^3.10.1",
```
Comment (WawrzenskiPatryk) [inline package.json:33]: `"eslint-plugin-tailwindcss": "3.10.1"` 🙏 
We've been following the convention of using exact versions of packages installed

**Review by WawrzenskiPatryk — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (zakrzaq) [reply package.json:33]: sure thing 👍🏻

**Comment by elmir-ismayilzada** [comment]
@zakrzaq Conflicts 👀

**Review by WawrzenskiPatryk — APPROVED** [review]
👍

**Pull request merged**